src/Sarif.Multitool.FunctionalTests/TestData/HashAlgorithmsMustBeUnique/NonUniqueHashAlgorithms.sarif

```diff
@@ -1,30 +1,12 @@
 {
-  "$schema": "http://json.schemastore.org/sarif-1.0.0",
-  "version": "1.0.0",
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
   "runs": [
     {
       "tool": {
         "name": "CodeScanner"
       },
-      "results": [
-        {
-          "ruleId": "TST0001",
```

Comment:
I haven't been ruthless about this, but in general, as I've updated the rules, I've tried to remove from the test files anything that's not needed for the test. Of course I leave anything required by the schema.